rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -962,6 +962,76 @@ public void updateCollections(Transaction rskTx) throws IOException {
         updateFederationCreationBlockHeights();
     }
 
+    protected void processSvpFundTransactionUnsigned(Transaction rskTx) throws IOException, InsufficientMoneyException {
+        Optional<Federation> proposedFederationOpt = federationSupport.getProposedFederation();
+        if (!proposedFederationOpt.isPresent()) {
+            String message = "Proposed federation should be present when processing SVP fund transaction.";
+            logger.warn(message);
+            throw new IllegalStateException(message);
+        }
+        Federation proposedFederation = proposedFederationOpt.get();
+
+        Coin spendableValueFromProposedFederation = bridgeConstants.getSpendableValueFromProposedFederation();
+        BtcTransaction svpFundTransactionUnsigned = createSvpFundTransaction(proposedFederation, spendableValueFromProposedFederation);
+
+        provider.setSvpFundTxHashUnsigned(svpFundTransactionUnsigned.getHash());
+        settleReleaseRequest(svpFundTransactionUnsigned, rskTx.getHash(), spendableValueFromProposedFederation);
```

Comment:
I believe we want to move away from the "release" name reference. It would be best if you used "pegout" instead.

Reply:
that name was a [suggestion](https://github.com/rsksmart/rskj/pull/2677#discussion_r1717490742) from marcos. Wdyt @marcos-iov ?

Reply:
I think release is a good name to cover the case for peg-out, migration, or fund tx.

We are releasing funds